simple-web-application/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>workshop</groupId>
+    <artifactId>streaming-data-workshop</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>workshop</groupId>
+  <artifactId>simple-web-application</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <main.verticle>workshop.playgroud.PlayWithTheCache</main.verticle>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-dependencies</artifactId>
+        <version>${vertx.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>workshop</groupId>
+      <artifactId>data-model</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-remote</artifactId>
```

Comment:
This is an uber jar, please do not depend on this since it creates havoc with dependencies. Uber jar dependencies are designed for environments with no dependency management system, e.g. ant.

Reply:
You should just need `infinispan-client-hotrod` and don't need version info.